movies-manager/src/FComponent.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+export function Bye(props) {
```

Comment:
Consistency violation. It should be named like FComponent.

Reply:
will improve in Task-5, with reason not to duplicate improvements